config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
    root 'welcome#index'
-   post '/login', to: 'session#create'
+
+   post '/login', to: 'sessions#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.